packages/react/src/forms/FormControl/FormControl.stories.tsx

```diff
@@ -296,12 +296,24 @@ export const CheckboxPlayground = args => {
 
 CheckboxPlayground.storyName = 'w/ Checkbox - Playground'
 
-export const RadioPlayground = () => (
+export const RadioPlayground = args => (
   <>
     <Stack direction={{narrow: 'vertical', regular: 'horizontal'}} gap="condensed" padding="condensed">
-      <FormControl>
+      <FormControl
```

Comment:
This is pretty much borrowed from the CheckboxPlayground implementation so I could re-use this component for the example with a validation message.